locales/en/messages.json

```diff
@@ -5,5 +5,11 @@
       "heading_proposed": "Move sent for review",
       "content": "Move reference<br><strong>{{moveReference}}</strong>"
     }
+  },
+  "create_allocation": {
+    "success": {
+      "heading": "Allocation requested",
+      "heading_proposed": "Move sent for review"
```

Comment:
You don't need this context. This was to support the different states when creating a move. It uses a context of `proposed` to display a different heading.